### PR TITLE
JobCN feature

### PR DESCRIPTION
**Pull request by huanzhang**
This pull requst is for JobCN feature #185

* add FFaker::JobCN module
* add job_cn/job_nouns
* add unit tests

### CODE REVIEW

File: ffaker.gemspec [hunk 1]
```diff
@@ -114,6 +114,7 @@ Gem::Specification.new do |s|
     lib/ffaker/data/job/job_adj
     lib/ffaker/data/job/job_noun
     lib/ffaker/data/job/job_prefix
+    lib/ffaker/data/job_cn/job_nouns
```
Comment (rocknruby) [inline ffaker.gemspec]: please revert your changes on gemspec files, this file is only for maintainer

Comment (rstacruz) [inline ffaker.gemspec]: also, maybe our gemspec should be looking more like this:

```rb
s.files = `git ls-files`.strip.split("\n")
```

Comment (rocknruby) [inline ffaker.gemspec]: @rstacruz there is a task for getting file-list dynamically. Its working fine :+1:

**Comment by huanzhang** [comment]
@rocknruby changes on gemspec has been reverted, please review.

**Comment by rstacruz** [comment]
:+1: from here

**Pull request merged**